Jenkinsfile

```diff
@@ -324,11 +319,11 @@ stage('Smoke Test') {
 def setEnvForDBModule(openShiftHost, openShiftToken, svcName, projName, mysqlUser, mysqlPwd) {
     try {
     sh """ 
-        oc env dc ${svcName} MYSQL_SERVICE_NAME=mysql -n ${projName} 2> /dev/null
-        oc env dc ${svcName} MYSQL_SERVICE_USERNAME=${mysqlUser} -n ${projName} 2> /dev/null
-        oc env dc ${svcName} MYSQL_SERVICE_PASSWORD=${mysqlPwd} -n ${projName} 2> /dev/null
-        oc env dc ${svcName} JAVA_APP_DIR=/deployments -n ${projName} 2> /dev/null
-        oc deploy ${svcName} --cancel -n ${projName} -n ${projName} 2> /dev/null
+        oc set env dc ${svcName} MYSQL_SERVICE_NAME=mysql -n ${projName} 2> /dev/null
+        oc set env dc ${svcName} MYSQL_SERVICE_USERNAME=${mysqlUser} -n ${projName} 2> /dev/null
+        oc set env dc ${svcName} MYSQL_SERVICE_PASSWORD=${mysqlPwd} -n ${projName} 2> /dev/null
+        oc set env dc ${svcName} JAVA_APP_DIR=/deployments -n ${projName} 2> /dev/null
+        oc rollout cancel dc ${svcName} -n ${projName} -n ${projName} 2> /dev/null
```

Comment:
I am assuming, these changes will work with the old versions, in case if the client has openshift version <3.7.

Reply:
According to openshift document, user will need at least OCP 3.3 to have this rollout command work, and oc set env should be fine from OCP 3.2